models/topics.models.js

```diff
@@ -6,3 +6,20 @@ exports.selectTopics = () => {
     return response.rows;
   });
 };
+
+exports.selectArticleId = (id) => {
+  return db
+    .query(
+      `SELECT articles.* FROM articles JOIN users ON users.username = articles.author WHERE articles.article_id = $1;`,
```

Comment:
We dont really need any info from the users table here as the author info is already added to the articles data